docs/how_to/use_sampler.rst

```diff
@@ -0,0 +1,144 @@
+    qc.measure_all()
+
+Initialize the ``Sampler``
+==========================
+
+Then, you need to create a :class:`~qiskit.primitives.Sampler` object.
+
+.. testcode::
+
+    from qiskit.primitives import Sampler
+
+    sampler = Sampler()
+
+Run and get results
+===================
+
+Now that you have defined ``sampler``, you can create a :class:`~.PrimitiveJob` (subclass of :class:`~qiskit.providers.JobV1`) with the
+:meth:`~qiskit.primitives.Sampler.run` method and, then, you can get the results (as a :class:`~qiskit.primitives.SamplerResult` object) with
+the results with the :meth:`~qiskit.providers.JobV1.result` method.
+
+.. testcode::
+
+    job = sampler.run(qc)
+    result = job.result()
+    print(result)
+
+.. testoutput::
+
+    SamplerResult(quasi_dists=[{0: 0.4999999999999999, 3: 0.4999999999999999}], metadata=[{}])
+
+Get the probability distribution
+--------------------------------
+
+From these results you can take the probability distributions with the attribute :attr:`~qiskit.primitives.SamplerResult.quasi_dists`.
+
+Even though there is only one circuit in this example, :attr:`~qiskit.primitives.SamplerResult.quasi_dists` returns a list of :class:`~qiskit.result.QuasiDistribution`\ s.
+Generally ``result.quasi_dists[i]`` would be the quasi-probability distribution of the ``i``-th circuit.
+
+.. testcode::
+
+    quasi_dist = result.quasi_dists[0]
+    print(quasi_dist)
+
+
+.. testoutput::
+
+    {0: 0.4999999999999999, 3: 0.4999999999999999}
+
+Probability distribution with binary outputs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you prefer to see the outputs as binary strings instead of decimal, you can use the
+:meth:`~qiskit.result.QuasiDistribution.binary_probabilities` method.
+
+.. testcode::
+    
+    print(quasi_dist.binary_probabilities())
+
+.. testoutput::
+
+    {'00': 0.4999999999999999, '11': 0.4999999999999999}
+
+Parameterized circuits with ``Sampler``
+=========================================
+
+The :class:`~qiskit.primitives.Sampler` primitive also has the option to include unbound parameterized circuits like the one below.
+You can also bind values to the parameters of the circuit like in :doc:`this guide <create_a_parameterized_circuit>` and follow the steps
+of the previous example.
+
+.. testcode::
+
+    from qiskit.circuit import Parameter
+
+    theta = Parameter('θ')
+    qc = QuantumCircuit(2)
+    qc.ry(theta, 0)
+    qc.cx(0,1)
+    qc.measure_all()
+    print(qc.draw())
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+            ┌───────┐      ░ ┌─┐   
+       q_0: ┤ Ry(θ) ├──■───░─┤M├───
+            └───────┘┌─┴─┐ ░ └╥┘┌─┐
+       q_1: ─────────┤ X ├─░──╫─┤M├
+                     └───┘ ░  ║ └╥┘
+    meas: 2/══════════════════╩══╩═
+                              0  1 
+
+The main difference from the previous case is that now you need to include the parameter values
+for which you want to evaluate the expectation value as a ``list`` of ``list``\ s of ``float``\ s.
```

Comment:
is this paragraph correct? Shouldn't be talking about sampling probabilities not expectation values?

Reply:
Good catch! Fixed in https://github.com/Qiskit/qiskit-terra/pull/8624/commits/ec0c28c0348eff3dfa05eb29ec7e229d47adfd46